mlir/include/mlir/Dialect/Linalg/Passes.td

```diff
@@ -75,6 +75,11 @@ def LinalgElementwiseOpFusionPass : Pass<"linalg-fuse-elementwise-ops"> {
   let dependentDialects = [
     "affine::AffineDialect", "linalg::LinalgDialect", "memref::MemRefDialect"
   ];
+  let options = [
+    Option<"introduceTensorEmpty", "introduce-empty", "bool",
```

Comment:
Maybe we go for `try-remove-outs-dependency` or the like? Introducing the `tensor.empty` is more of a way to achieve what the pattern wants.

Reply:
I would go for something in-between like `replace-outs-by-tensor-empty`, I really like on explicitly it is and I'm hoping one is only going to revisit this when you want/don't want a tensor empty.

Reply:
I settled on replace-outs-dependency